index.bs

```diff
@@ -313,16 +327,24 @@ Abstract Operations {#abstract-opertaions}
 <div algorithm="construct a magnetometer object">
 
     : input
+    :: |magnetometer_interface|, a {{Magnetometer}} [=interface=] [=identifier=] or
+       an {{UncalibratedMagnetometer}} [=interface=] [=identifier=].
     :: |options|, a {{MagnetometerSensorOptions}} object.
     : output
     :: A {{Sensor}} object.
```

Comment:
a {{Magnetometer}} object?

Reply:
I think Sensor object is correct as an output, or use something like:

`:: Instance of an |magnetometer_interface|`

Reply:
Output can be either `Magnetometer` or `UncalibratedMagnetometer`, both of which are also instances of a `Sensor`. Not sure how to best spec this, to be explicit maybe:
```
:: A {{Magnetometer}} or {{UncalibratedMagnetometer}} object.
```

Reply:
Fix https://github.com/w3c/magnetometer/pull/40/commits/474792adcb75b961267abd573bb41574b3a6b541